buddychatcli/ParticipantDataGenerator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using CommandLine;
+using System.IO;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("BuddyChatCLI.test")]
+namespace BuddyChatCLI
+{
+    [Verb("CreateParticipantData", HelpText = "Creates Participant Data combining the existing data so far with the new signup data from the csv files into a new csv file")]
+    public class ParticipantDataGenerator
+    {
+        [Option(shortName: 'e',
+                longName: "existingDataFilePath",
+                Required = true,

```

Comment:
If required is true, then does it still let the program default to current directory?